styles.css

```diff
@@ -0,0 +1,115 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f4f5f7;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+}
+
+.header {
+  height: 100px;
+}
+
+.header-container {
+  height: 92px;
+  width: 100%;
+  padding-left: 5%;
+  padding-right: 5%;
+  position: fixed;
+  background-color: white;
+}
+
+.status-bar {
+  height: 44px;
+}
+
+.topbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-top: 10px;
+  list-style: none;
+}
+
+.app-bar {
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 12px;
+}
+
+.logo {
+  font-size: 20px;
+  font-weight: 600;
+  color: #6070ff;
+}
+
+.fa-bars {
+  font-size: 1.5em;
+  color: #6070ff;
+}
+
+.nav-items {
+  display: none;
+}
+
+.hero-section {
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-color: #fff;
+  background-size: cover;
+  background-position: center;
+  padding-top: 70px;
+  height: 718px;
+  border-bottom-left-radius: 120px;
+}
```

Comment:
- [ ] I **HIGHLY** recommend you not to fix the height of your container ⛏️ If the content inside the section exceeds this height, it will overflow and cause the section to appear larger than intended. Which is the current case. It is **overflowing** 🌊 🌊 
To amend that, I recommend you to remove the fixed height for the **_hero-section_**, and allow it to expand naturally based on the content inside.
`height: auto`
You can adjust the padding and margins of the content inside to fit within the given height.
For example, adjusting the padding on **_hero-container_** class 😁